app/lib/communicator/communicator.js

```diff
@@ -223,6 +225,41 @@ function getSubmissionUrl( server ) {
     return ( server.lastIndexOf( '/' ) === server.length - 1 ) ? `${server}submission` : `${server}/submission`;
 }
 
+/**
+ * @param {string} value
+ */
+const sanitizeHeader = ( value ) => {
+    return value
+        .trim()
+        .replace( /\s+/g, ' ' )
```

Comment:
Maybe this replace and trim just need to be swapped?

Reply:
Oh no what am I saying I'm deleting this. The newline ends up in the middle of the string.